allennlp/nn/decoding/decoder_trainers/expected_risk_minimization.py

```diff
@@ -30,11 +30,18 @@ class ExpectedRiskMinimization(DecoderTrainer[Callable[[StateType], torch.Tensor
         al. do this in their work.
     max_decoding_steps : ``int``
         The maximum number of steps we should take during decoding.
+    max_num_decoded_sequences : ``int``, optional (default=1)
+        Maximum number of sorted decoded sequences to return. Defaults to 1.
     """
-    def __init__(self, beam_size: int, normalize_by_length: bool, max_decoding_steps: int) -> None:
+    def __init__(self,
+                 beam_size: int,
+                 normalize_by_length: bool,
+                 max_decoding_steps: int,
+                 max_num_decoded_sequences: int = 1) -> None:
         self._beam_size = beam_size
         self._normalize_by_length = normalize_by_length
         self.max_decoding_steps = max_decoding_steps
```

Comment:
Nit: Why is this public while the other three are private?  I have no strong preference here, if you want it to public, it just seems inconsistent.

Reply:
Changed it, thanks!